jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>this reads now as a long list of different tools, can we group them and explain the underlying concept and then state "tool A and tool B are examples for these concepts, we now show how to use it..". The lineage tracing chapter does this super nicely! maybe check out and see how we can come up with a similar structure here: https://www.sc-best-practices.org/trajectories/lineage_tracing.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I'm not sure that we can find a way to group them. They are just different methods in the field but they try to do a similar thing. The only different algorithms are AutoGeneS, which focuses on the feature selection, and CPM, which mainly does cell-state deconvolution and not focusing on cell-type deconvolution as the rest. We can group the first 4 and then add the last two as another group but I don't know if this is better.</p>

Reply:
<p>Okay now?</p>

Reply:
<p>"During the last decade, many benchmarking papers have been published, highlighting the advantages and disadvantages of different cell deconvolution methods {cite}<code>ShenOrr2013,Cobos2020,Jin2021,Nadel2021</code>. Here is a list of some of the prominant methods including CibersortX, MuSiC, dtangle, DWLS, xCell and Scaden. It should be mentioned that there are several other deconvolution methods proposed recently that have not been studied in those benchmarking papers. Here we discuss two of them: AutoGeneS and CPM. "</p><p> </p><ol><li>Please state very clearly what the results of the benchmarks are. Do they agree? If yes, what are their recommendations? If not, where and why do they disagree?</li><li>Therer is not a single chapter in the book that just lists methods. We usually introduce different approaches (which is why Anna asked for the grouping) and pick one tool per approach. Here, it sounds like there's one mainstream approach and we show MuSiC because it's not webbased and actually accessible. There's no need to introduce all methods in detail like you did, but citing them in the context of the benchmarking results is good enough.</li><li>We can mention novel tools like CPM/AutogeneS and alternative approaches like the enrichment + non-linear methods in a "new directions" section at the end of the chapter like https://www.sc-best-practices.org/trajectories/lineage_tracing.html#new-directions did really nicely. For consistency, we will insist on keeping the style homogeneous.</li><li>You write "highly used deconvolution method. " a couple of times. The correct expression is "widely used" :)</li></ol>

Reply:
<p>3 and 4: Sure, I can do that.</p><p>2: This is the truth. We cannot focus on one method and say it's the best. We won't do that. That is why in this chapter it is important to list many methods. I can remove all the details about them if you prefer but it's important to list many of them. The deconvolution chapter is not the same as other chapters. There is no clear winner method in the field (for many reasons).</p><ol><li>Sorry but I don't think we should do that.  This would be just the wrong thing to do in this field.</li></ol>

Reply:
<p>Moved CPM and AutoGeneS to the new directions section</p>